include/pybind11/numpy.h

```diff
@@ -1551,6 +1551,7 @@ struct vectorize_helper {
             "pybind11::vectorize(...) requires a function with at least one vectorizable argument");
 
 public:
+    // NOLINTNEXTLINE(bugprone-forwarding-reference-overload) //TODO FIX BEFORE MERGE
```

Comment:
@rwgk This is the prefect forwarding code that I was struggling with fixing the error for.

Reply:
Okay, so we do need to the copy and move constructors after all... Hmm... so we need to do some SFINAE acrobatics as described in https://moodle.ufsc.br/pluginfile.php/2377667/mod_resource/content/0/Effective_Modern_C__.pdf Item 25-26 Still struggling with it though.